config.py

```diff
@@ -156,15 +156,15 @@ def get_policy(algorithm, policy_arg, action_size, model, uncertainty):
     return {"policy": policy, "test_policy": policies.GreedyPolicy(action_size, model)}
 
 
-def get_uncertainty(algorithm, model, action_size, variance_prior):
+def get_uncertainty(algorithm, model, action_size, variance_prior, varuance_likelihood):
     if algorithm.startswith('sf'):
         uncertainty = OnlineVariance(
-            model.feature_size, train_featuriser=model.local_features,
-            test_featuriser=model.global_features, prior_variance=variance_prior, bias=False)
+            model.feature_size, train_featuriser=model.local_features, test_featuriser=model.global_features,
+            prior_variance=variance_prior, likelihood_variance=varuance_likelihood, bias=False)
```

Comment:
Fix spelling varuance

Reply:
Oops, thanks for fixing my bad spelling.